source/manage/telemetry.rst

```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
 - Date and time of the last check for security updates
 - The location of the Amazon Cloudfront server used for telemetry data
 
-To opt out, disable the feature in **System Console > Environment > SMTP**. When the feature is disabled, you will not receive any security alerts.
+Opt out
+~~~~~~~
+
+To opt out, you can disable this security update check feature for self-hosted deployments in the System Console by going to **Environment > SMTP > Enable Security Alerts**. See the `enable security alerts </configure/environment-configuration-settings.html#enable-security-alerts>`__ documentation for details. When this feature is disabled, you will not receive any security alerts.
 
 Error and diagnostics reporting feature
 ---------------------------------------
 
 Mattermost error and diagnostic data is collected for the following purposes:
+
 - To add improvements that are specific to your usage and deployment patterns, including identifying security and reliability issues.
 - To analyze and fix software problems.
 - To help improve the quality of Mattermost software and related services.
 - To make design decisions for future releases.
 
 .. note:: 
 
-  Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. From Mattermost v5.23 the segment endpoint is being deprecated in favor of ``https://pdat.matterlytics.com``, a custom Rudder domain. To opt out, you can disable the feature in **System Console > Environment > Logging > Enable Diagnostics and Error Reporting**. Starting in Mattermost v5.26, when this feature is enabled, any 500 errors will be automatically sent to the Mattermost-hosted `Sentry <https://sentry.io/welcome/>`_ endpoint.
+  Error and diagnostic reporting is sent by the client to the endpoint ``api.segment.io``. From Mattermost v5.23, the segment endpoint is being deprecated in favor of ``https://pdat.matterlytics.com``, a custom Rudder domain. From Mattermost v5.26, when this feature is enabled, any 500 errors will be automatically sent to the Mattermost-hosted `Sentry <https://sentry.io/welcome/>`_ endpoint.
```

Comment:
We don't use the word "client" often in our docs. Should we change this to "server"?

Reply:
I recommend a simplification from "client" to "Mattermost".